### DISCUSSION TITLE
Growing the Community 🚀

### DISCUSSION THREAD

**Discussion by yezz123** [piccolo-orm/piccolo#341] [General]
@dantownsend I think it's time to give the community the place to share the way how piccolo can be addons now and in the future, maybe creating a discord server or a Slack channel can help it grow fast and have a big community. 🚀

**Comment by dantownsend** [comment]
@yezz123 Yes, it's a good idea. I looked at some options a while back, and decided to go with Github Discussions as it was quite easy.

I know a lot of people use chat apps though. Discord in particular seems really popular. Do you have a preference?

**Reply by yezz123** [reply]
Literally, I prefer discord looks more compatible with devs and what devs can do on it, slack look professional and not supportive for a lot of features that discord share with the community.

Github discussion is great also, but we can let it be more technical, for some issues or some ideas that can be released in the future.

**Reply by AliSayyah** [reply]
I think lots of people are not very familiar with Github discussion, but discord is very popular and it's easier to manage a big crowd and sort things out.

**Reply by yezz123** [reply]
Somehow yes using a platform for sharing ideas and ways of thinking during the development of this package, that's why its look better to use discord ✨
